addons/mail/models/mail_thread.py

```diff
@@ -4033,7 +4033,7 @@ def _message_update_content(self, message, body, attachment_ids=None,
         if strict:
             self._check_can_update_message_content(message.sudo())
 
-        msg_values = {'body': body} if body is not None else {}
+        msg_values = {'body': body}
```

Comment:
Why this change ? Anyway it does not respect the docstring (and I see no test for this change ?).

Reply:
I am dropping this change for now. The reason of the change is, in the current webClient part, we always send the body in updating message content. It's better to change the client part in a follow-up PR.

Reply:
Yep, better give only the values that change, like a call to ``write``, it avoids unnecessary computation / update / ...